packages/dds/tree/src/test/memory/tree.spec.ts

```diff
@@ -33,28 +39,65 @@ class RootNodeSchema extends builder.object("root-item", {
 	child: builder.optional(ChildNodeSchema),
 }) {}
 
-function createLocalSharedTree(id: string): TreeView<typeof RootNodeSchema> {
-	const sharedTree = SharedTree.create(
+const initialState = {
+	child: {
+		propertyOne: 128,
+		propertyTwo: {
+			itemOne: "",
+		},
+		propertyThree: new Map([["numberOne", 1]]),
```

Comment:
We now support building map nodes from iterables of key value pairs, so you can do just 
```suggestion
		propertyThree: [["numberOne", 1]],
```

It is also valid to do:
```suggestion
		propertyThree: {numberOne: 1},
```

Reply:
Updated with second suggestion